echopop/__init__.py

```diff
@@ -1,5 +1,6 @@
 from .survey import Survey
+from .utils import operations
 
-__all__ = ["Survey"]
+__all__ = ["Survey", "operations"]
```

Comment:
why do you need to expose `operations` at the root level when importing the package? It is not currently used in the example notebook.